Samples/Samples.iOS/Info.plist

```diff
@@ -55,8 +55,10 @@
 	<string>1.0</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Access to your location is required for cool things to happen!</string>
-  <key>NSCalendarsUsageDescription</key>
-  <string>Access to view and create calendar events</string>
+    <key>NSCalendarsUsageDescription</key>
+    <string>Access to view and create calendar events</string>
+    <key>NSRemindersUsageDescription</key>
```

Comment:
Just confirming we need this for writing basic calendar events?

Reply:
Ahh I suppose we wouldn't need it yet, I believe, happy to remove it at this stage if we're not planning on tackling reminders in the near future/getting a release out inbetween

Reply:
Yes, we should only include absolute minimum permissions we need for what we're doing

Reply:
Altered accordingly